tools/repl/swift/CMakeLists.txt

```diff
@@ -1,50 +1,10 @@
-set(LLVM_NO_RTTI 1)
+# Set the correct rpath to locate libswiftCore
+set(CMAKE_EXE_LINKER_FLAGS "-Wl,-rpath \
+    -Wl,${CMAKE_BINARY_DIR}/${LLDB_FRAMEWORK_INSTALL_DIR}/${LLDB_FRAMEWORK_RESOURCE_DIR}/Swift/macosx")
```

Comment:
Should we move this, and the add_lldb_tool invocation, into the `if (LLDB_BUILD_FRAMEWORK)`?

Reply:
Probably. I'll do this tomorrow.